scripts/cleanup_failed_registrations.py

```diff
@@ -50,7 +50,8 @@ def remove_failed_registrations(dry_run=True):
                 f.registered_from,
                 f,
                 f.creator,
-                f.archive_job.target_info()
+                f.archive_job.target_info(),
+                email_users=False
```

Comment:
As per the ticket, we only want to be notified on Desk about stuck registrations; there should be no db writes. So rather than adding a new parameter to `handle_archive_fail` (which appears to do db writes), why not just remove the call to `handle_archive_fail` and send the desk emails?